PlaybackLab/MultiplePlayersScrollingViewSampleApp/MultiplePlayersScrollingViewSampleApp/ViewControllers/MultiplePlayerViewController.swift

```diff
@@ -0,0 +1,257 @@
+//
+//  MultiplePlayerViewController.swift
+//  MultiplePlayersScrollingViewSampleApp
+//
+//  Copyright © 2019 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class MultiplePlayerViewController: UIViewController {
+  
+  let options = OptionDataSource.options
+  
+  var titleLabel: UILabel!
+  var collectionView: UICollectionView!
+  
+  var sharedPlayer: OOSkinViewController!
```

Comment:
We may do the same with `private lazy var` to avoid optionality. It's a good practice
We may also introduce an auxiliary variable in order to keep them simple. Like this:
```
  private lazy var player: OOOoyalaPlayer = {
    let playerSelectionOption = options[0]
    var player: OOOoyalaPlayer
    var apiKey: String
    var apiSecret: String

    if playerSelectionOption.embedTokenGenerator != nil {
      if let basicEmbedTokenGenerator = playerSelectionOption.embedTokenGenerator as? BasicEmbedTokenGenerator {
        apiKey = basicEmbedTokenGenerator.apiKey
        apiSecret = basicEmbedTokenGenerator.apiSecret
      } else {
        // If you're not using the BasicEmbedTokenGenerator provided in the example,
        // supply your own API_KEY and API_SECRET
        apiKey = "API_KEY"
        apiSecret = "API_SECRET"
      }

      let options = OOOptions()!
      // For this example, we use the OOEmbededSecureURLGenerator to create the signed URL on the client
      // This is not how this should be implemented in production -
      // In production, you should implement your own OOSecureURLGenerator
      // which contacts a server of your own, which will help sign the url with the appropriate API Key and Secret
      options.secureURLGenerator = OOEmbeddedSecureURLGenerator(apiKey: apiKey,
                                                                secret: apiSecret)

      player = OOOoyalaPlayer(pcode: playerSelectionOption.pcode,
                              domain: playerSelectionOption.domain,
                              embedTokenGenerator: playerSelectionOption.embedTokenGenerator,
                              options: options)

    } else {
      player = OOOoyalaPlayer(pcode: playerSelectionOption.pcode,
                              domain: playerSelectionOption.domain)
    }

    player.actionAtEnd = .pause
    return player
  }()
```
and
```
  
  private lazy var sharedPlayer: OOSkinViewController = {
    let jsCodeLocation = Bundle.main.url(forResource: "main",
                                         withExtension: "jsbundle")!

    let skinOptions = OOSkinOptions(discoveryOptions: nil,
                                    jsCodeLocation: jsCodeLocation,
                                    configFileName: "skin",
                                    overrideConfigs: nil)

    let skinViewController = OOSkinViewController(player: self.player,
                                                  skinOptions: skinOptions,
                                                  parent: UIView())

    skinViewController.isAutoFullscreenWithRotatedEnabled = true
    skinViewController.view.translatesAutoresizingMaskIntoConstraints = false
    return skinViewController
  }()
```

After these changes we don't need following lines in `viewDidLoad`
```
    guard let playerSelectionOption = options.first else { return }
     initPlayer(playerSelectionOption)
```